joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50 

right_front_controller:
  type: velocity_controllers/JointVelocityController
  joint: right_front_joint 
  pid: {p: 10.0, i: 0.01, d: 0.0}

left_front_controller:
  type: velocity_controllers/JointVelocityController
  joint: left_front_joint
  pid: {p: 10.0, i: 0.01, d: 0.0}

right_back_controller:
  type: velocity_controllers/JointVelocityController
  joint: right_back_joint  
  pid: {p: 10.0, i: 0.01, d: 0.0}

left_back_controller:
  type: velocity_controllers/JointVelocityController       
  joint: left_back_joint  
  pid: {p: 10.0, i: 0.01, d: 0.0}


# 驱动器接受速度指令即 velocity_controllers/JointVelocityController  接受速度指令，输出速度（这里PID无效）
# 驱动器接受力矩指令即 effort_controllers/JointVelocityController    接受力矩指令，输出速度   

